utilities/helper/accelerate.py

```diff
@@ -168,14 +168,15 @@ def daemonize_self(module, password, port, minutes, pid_file):
             vvv("exiting pid %s" % pid)
             # exit first parent
             module.exit_json(msg="daemonized accelerate on port %s for %s minutes with pid %s" % (port, minutes, str(pid)))
-    except OSError, e:
-        log("fork #1 failed: %d (%s)" % (e.errno, e.strerror))
-        sys.exit(1)
+    except OSError:
+        e       = get_exception()
+        message = "fork #1 failed: %d (%s)" % (e.errno, e.strerror)
+        module.fail_json(message)
```

Comment:
@Hypermanzer I think we want to use message as a kwarg:  ```module.fail_json(msg=message)```
@bcoca does this look right to you?  Logically, I think that it's correct.  If the fork fails then we're still in the first parent and so we want to use module.fail_json() to return json output.  But I don't write forking code very often so I'd appreciate a second set of eyes to make sure that's the correct thing to do here.